exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -279,7 +279,10 @@ public ProtobufLengthDecoder getDecoder(BufferAllocator allocator, OutOfMemoryHa
 
   @Override
   public void close() throws IOException {
-    Closeables.closeQuietly(authenticator);
-    super.close();
+    try {
```

Comment:
why are you wrapping this inside a `try` clause ? do you suspect authenticator.close() can throw other exceptions other than `IOException` ?